code/API_definitions/carrier_billing.yaml

```diff
@@ -1142,12 +1197,55 @@ components:
       description: Requested number of resources to be provided in response
       schema:
         type: integer
-    Seek:
-      name: seek
+    StartDate:
+      description: Initial `paymentCreationDate` for running the query, in ISO-8601 extended local date format. Time-offset from UTC may be used to match local OB time
```

Comment:
We have probably an small issue ahead with using ISO-8601 mention in this API are we are moving for RFC3339 - source: https://github.com/camaraproject/Commonalities/issues/14

Perhaps we should switch to RF339 mention

Reply:
I agree that besides the meaning of the field, make a reference to RFC in the terms of the output of that Commonalities issue. 

I have provided a suggestion. Once we have a common wording described in commonalities, I will be adding here for "date" fields